addons/buddypress/buddypress.php

```diff
@@ -587,7 +587,7 @@ public function remove_comment_notify( $comment ) {
 			return;
 		}
 
-		if ( $comment->comment_ID ) {
+		if ( isset( $comment->comment_ID ) && $comment->comment_ID ) {
```

Comment:
@mebishalnapit check for `$comment` instead, we don't need `isset` here.

Reply:
@rahularyan It will not work if we check for `$comment` since this method is also hooked to the `ap_trash_answer` action hook: https://github.com/anspress/anspress/blob/fa4c4aeb9dc89c4859d0f2669624b8acdf372860/addons/buddypress/buddypress.php#L68 https://github.com/anspress/anspress/blob/fa4c4aeb9dc89c4859d0f2669624b8acdf372860/includes/hooks.php#L318 which has the first argument as post id so `$comment->comment_ID` will not exists for the first condition available, so we will need to check for isset.